src/main/scala/uk/gov/nationalarchives/tdr/api/service/ConsignmentService.scala

```diff
@@ -65,4 +66,14 @@ class ConsignmentService(
   def getConsignmentParentFolder(consignmentId: UUID): Future[Option[String]] = {
     consignmentRepository.getParentFolder(consignmentId)
   }
+
+  private def convertRowToConsignment(row: ConsignmentRow): Consignment = {
+    Consignment(
+      row.consignmentid,
+      row.userid,
+      row.seriesid,
+      Some(row.datetime.toLocalDateTime),
```

Comment:
Just wondering - if this value is non-nullable in the database, could it be non-optional in the Consignment model too?